source/grid/grid_tools.cc

```diff
@@ -2662,7 +2662,7 @@ namespace GridTools
     static inline std::bitset<3> lookup (const MATCH_T &)
     {
       // The 1D case is trivial
-      return 4; // [true ,false,false]
+      return 1; // [true ,false,false]
```

Comment:
On Tue, Sep 22, 2015, at 13:30 CDT, Martin Kronbichler <notifications@github.com> wrote:

>> @@ -2662,7 +2662,7 @@ namespace GridTools
>>      static inline std::bitset<3> lookup (const MATCH_T &)
>>      {
>>        // The 1D case is trivial
>> -      return 4; // [true ,false,false]
>> +      return 1; // [true ,false,false]
>
> As I said, I'm not sure that my patch is right - I merely tried to
> reproduce what 2D/3D give for the standard case

It is correct. The least significant bit is determined by bitset[0], the
most significant bit is bitset[2] - I shot myself in the foot by
assuming big-endian (with respect to the index).

Best,
Matthias